plugins/src/android/OptimoveInitProvider.java

```diff
@@ -74,4 +93,9 @@ private String getStringConfigValue(String packageName, Resources resources, Str
 
         return resources.getString(resId);
     }
+
+    @Nullable
```

Comment:
it's a style thing probably, but this annotation applies to the return type, so, mb nicer to put it before that , mm, for consistency? I think in android we have it that way